### PR TITLE
edit filter page according to design

### PR DESCRIPTION
**Pull request by bugragoksu**
image

### CODE REVIEW

File: lib/pages/demands_page/widgets/demand_filter_popup.dart [hunk 1]
```diff
@@ -77,113 +80,148 @@ class _DemandFilterDrawerState extends State<DemandFilterDrawer> {
...
-            const SizedBox(width: 4),
-          ],
-          IconButton(
-            onPressed: _onSave,
-            icon: const Icon(Icons.check_rounded),
-          )
-        ],
+        ),
       ),
-      body: SingleChildScrollView(
```
Comment (easeccy) [inline lib/pages/demands_page/widgets/demand_filter_popup.dart:99]: This page have to be scrollable.

**Review by easeccy — CHANGES_REQUESTED** [review]
Fixes needed

File: lib/pages/demands_page/widgets/demand_filter_popup.dart (hunk 1 above)
Comment (bugragoksu) [reply lib/pages/demands_page/widgets/demand_filter_popup.dart:99]: I'm not sure this page should be scrollable, maybe the cips could be scrollable but we don't need to scroll all the page
image

**Comment by easeccy** [comment]
@erolkaftanoglu possible stale PR, please reassign to someone else ASAP.

**Comment by erolkaftanoglu** [comment]
@easeccy we talked with @bugragoksu,  category area should be scrollable because be'cuz we don't know how much category count. he can fix right know

**Review by easeccy — APPROVED** [review]

**Pull request merged**